UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -244,19 +246,22 @@ private async Task<bool> ProcessBatchFile(IEnumerable<string> uri, string downlo
 
         private static async Task CopyFileToFolder(HttpResponseMessage httpResponse, string path)
         {
-            Stream stream = await httpResponse.Content.ReadAsStreamAsync();
-            FileStream outputFileStream = new FileStream(path, FileMode.Create, FileAccess.ReadWrite); 
-            stream.CopyTo(outputFileStream);
-        }      
+            using (Stream stream = await httpResponse.Content.ReadAsStreamAsync())
+            {
+                using (FileStream outputFileStream = new FileStream(path, FileMode.Create, FileAccess.ReadWrite))
+                {
+                    stream.CopyTo(outputFileStream);
+                }
+            }
+        }
 
         public async Task<bool> DownloadReadMeFile(string readMeFilePath, string batchId, string exchangeSetRootPath, string correlationId)
         {
             string payloadJson = string.Empty;
             var accessToken = await authTokenProvider.GetManagedIdentityAuthAsync(fileShareServiceConfig.Value.ResourceId);
-         
-            string fileName = fileShareServiceConfig.Value.ReadMeFileName;            
-            string file = Path.Combine(exchangeSetRootPath, fileName);           
-            CheckCreateFolderPath(exchangeSetRootPath);              
+            string fileName = fileShareServiceConfig.Value.ReadMeFileName;
+            string file = Path.Combine(exchangeSetRootPath, fileName);
```

Comment:
Should the variable file be named filePath?

Reply:
Done.